index.html

```diff
@@ -2,12 +2,275 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite App</title>
+    <title>VueConf Toronto</title>
+
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/favicon/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/favicon/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/favicon/site.webmanifest" />
+    <script type="module" src="./src/main.js"></script>
   </head>
   <body>
-    <div id="app"></div>
-    <script type="module" src="/main.js"></script>
+    <header class="header">
+      <div class="toolbar">
+        <div
+          class="
+            d-none d-md-flex
+            justify-md-end
+            align-md-center
+            gap-md-1
+            primary
+            container
+          "
+        >
+          <a href="#" aria-label="Facebook"
+            ><span class="icon icon-facebook" aria-hidden></span
+          ></a>
+          <a href="#" aria-label="Twitter"
+            ><span class="icon icon-twitter" aria-hidden></span
+          ></a>
+
+          <p class="white--text">English</p>
+
+          <p class="white--text">Signup</p>
+
+          <p class="white--text">Login</p>
+        </div>
+      </div>
+
+      <div class="d-md-flex justify-md-space-between align-center container">
+        <a class="d-none d-md-flex" href="/" aria-describedby="logo">
+          <span class="d-none" id="logo">Logo</span>
+          <img src="/logo.svg" alt="Logo image" />
+        </a>
+
+        <nav
+          class="
+            d-none d-md-flex
+            flex-column flex-md-row
+            justify-md-space-around
+            align-md-center
+            gap-3
+            menu
+          "
+        >
+          <button
+            class="d-flex d-md-none btn btn-close"
+            type="button"
+            aria-label="Mobile close button"
+          >
+            <span class="icon icon-close icon-2x" aria-hidden></span>
+          </button>
+
+          <ul
+            class="
+              d-none
+              flex-column
+              align-center
+              gap-1
+              d-md-flex
+              flex-md-row
+              justify-md-space-around
+              gap-md-2
+              nav-items
+            "
+          >
+            <li class="nav-item">
+              <a class="secondary--text nav-link" href="#home">About</a>
```

Comment:
- [ ] One of this projects requirements was to build 2 pages, the home page and the about page, as you can see here:
![image](https://user-images.githubusercontent.com/66967274/142069186-286eb326-04bb-416e-b092-e03e065852e9.png)

- [ ] This link should be redirecting to the `about` page and not the `home`, keep in mind that it should contain the following sections:
- [ ] **Top section**: centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
- [ ] **Middle section**: a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo
- [ ] **Bottom section**: a centered column that includes a section header, a divider, a line of text, and two images with text centered over them.
> Keep in mind that the in the bottom section you should have different styles for mobile and desktop versions.
> The header and footer should also be in this page.
- [ ] It should look something like this: 

![image](https://user-images.githubusercontent.com/66967274/142068164-f01ddd46-3d24-406b-9c13-3a2d8c8df76a.png)

Reply:
Fixed